test/rekt/resources/trigger/trigger.go

```diff
@@ -184,3 +187,27 @@ func Install(name, brokerName string, opts ...manifest.CfgFn) feature.StepFn {
 func IsReady(name string, timing ...time.Duration) feature.StepFn {
 	return k8s.IsReady(GVR(), name, timing...)
 }
+
+func WithNewFilters(filters []eventingv1.SubscriptionsAPIFilter) manifest.CfgFn {
+	jsonBytes, err := json.Marshal(filters)
+	if err != nil {
+		panic(err)
+	}
+
+	yamlBytes, err := yaml.JSONToYAML(jsonBytes)
+	if err != nil {
+		panic(err)
+	}
+
+	filtersYaml := string(yamlBytes)
+
+	lines := strings.Split(filtersYaml, "\n")
+	out := make([]string, 0, len(lines))
+	for i := range lines {
+		out = append(out, "    "+lines[i])
```

Comment:
Is that a `tab`? On my end there are 4 spaces.

Reply:
Yeah I just put 4 spaces

Reply:
Agree with you on using the space. All those yaml templates are using 2 spaces for indentation.